recipes/miktex-portable/meta.yaml

```diff
@@ -0,0 +1,34 @@
+{% set version = "2.9.5857" %}
+
+package:
+  name: miktex-portable
+  version: {{ version }}
+
+source:
+  fn: miktex-portable-{{ version }}.exe
+  url: http://mirrors.ctan.org/systems/win32/miktex/setup/miktex-portable-{{ version }}.exe
+  sha1: 2b50fd29b3683dbe769dd1891e075ef8941658f7
+
+build:
+  skip: True  # [not win]
+  number: 0
+  binary_relocation: false
+
+requirements:
+  build:
+    - 7za
+
+test:
+  commands:
+    - pdflatex --help
+    - mpm --help
+
+about:
+  home: http://miktex.org/
+  license: GPL
```

Comment:
Ok, it gets worse: https://anonscm.debian.org/cgit/debian-tex/texlive-bin.git/tree/debian/copyright

This is for texlife, but its basically the same as the one in the miktex homepage... seeing this, the easiest way is to add `various`...